### DISCUSSION TITLE
Refreshing of profile prior to connecting

### DISCUSSION THREAD

**Discussion by joanandk** [OpenVPN/openvpn3#341] [General]
Hi,
I am trying to create a dynamic profile for users. Is there any possiblity to configure openvpn 3 connect to download the profile each time prior to connecting to the server? 
I am trying to create an intelligent load sharer which consists of the following:
- a webserver which serves profiles, they are dynamically created on request from the client. The server checks how many VPN nodes are available and offers three of them via a profile.
- some nodes (created dynamically as the demand rises and shutdown as there is no need for them, via socket activation)
This method would be much more efficient and flexible than load sharing with iptables or DNS round robin.

At the moment, the client receives a profile and keeps it for ever (tried via "Rest API" and Webauth). May be I am missing something to trigger renewal of the profile. 

Thanks and BR

**Comment by dsommers** [comment]
There are currently no support for dynamically fetch and update a configuration profile.  Since this is not a limitation in the OpenVPN 3 Core Library, but the OpenVPN Connect client - I would suggest raising this as a feature request via https://support.openvpn.com/ .... All OpenVPN Connect users can use the support.

**Reply by joanandk** [reply]
> I would suggest raising this

Just send a request. Thanks. Hope they are willing to implement this idea.
